store/api.go

```diff
@@ -567,6 +568,27 @@ func (s *Store) TimeLeft(name string) (seconds float64) {
 	return s.timeLeft(name)
 }
 
+// TsdbTimeSeries returns the time series with given name from given endpoint.
+//
+// If no such metric exists for given endpoint, TsdbTimeSeries returns false
+// instead of true for the second argument.
+//
+// name is the name of the time series; endpointId identifies the endpoint.
+// start and end are seconds since Jan 1, 1970 and denote the start time
+// inclusive and the end time exclusive.
+//
+// TsdbTimeSeries only works on time series with numeric values. For other
+// time series, it returns nil, false as if the time series were not founc.
```

Comment:
Also, it's probably clearer to say "it returns (nil, false)" rather than say "it returns nil, false"